### ISSUE TITLE
ErrorLog Console Minecrfat FORGE Server 1.16.5 Mod ver. 0.9.3

### ISSUE THREAD

**Issue by APaskowski1994** [dainxt/AncientClasses#1]
Here the Warning:
```
[11:03:23] [Thread-1/WARN] [ne.mi.fm.co.ConfigFileTypeHandler/CONFIG]: Configuration file /Users/andrew/Desktop/Minecraft Server/Server/./Better_Minecraft/serverconfig/ancientclasses-server.toml is not correct. Correcting
```

Got a post of the Error in a nother mod maby helpfull.
https://github.com/AppliedEnergistics/Applied-Energistics-2/issues/4646

Thanks for support.
Grettings
Andrew

**Comment by dainxt** [comment]
Hello, did you manipulate the configuration file? If so, which parameters you modified.

**Comment by APaskowski1994** [comment]
hi there no i didnt touched the config file. I also deleted the config file and the .bak files and create new ones.

**Comment by APaskowski1994** [comment]
And the .toml file looks like:
```

#Ancient Classes Mod Configuration Settings
[general]
	#Enable this if you want to apply your branches configuration
	"Enable Custom Values" = false

#Branches related configuration, All values must be > 0
[branches]

	[branches.stone_branch]

		[branches.stone_branch.boost_ability]
			maxCost = [8, 6, 4]
			maxCooldown = [500, 300, 240]
			maxDuration = [80, 100, 120]
			maxCharges = [1, 1, 2]

		[branches.stone_branch.protect_ability]
			maxCost = [4, 4, 4]
			maxCooldown = [1200, 1000, 800]
			maxDuration = [180, 240, 300]
			maxCharges = [5, 7, 9]
			maxRange = [1, 2, 3]
			maxDamageReduction = [0.06, 0.12, 0.18]

		[branches.stone_branch.dash_ability]
			maxCost = [4, 4, 4]
			maxCooldown = [400, 360, 320]
			maxDuration = [20, 20, 20]
			maxVelocity = [1.0, 1.25, 1.5]

		[branches.stone_branch.cover_ability]
			maxCost = [6, 8, 10]
			maxCooldown = [3600, 1800, 900]
			maxDuration = [400, 400, 400]
			maxDomeRange = [2, 3, 4]
			maxBridgeRange = [8, 12, 16]

	[branches.diamond_branch]

		[branches.diamond_branch.condense_magic_ability]
			maxCost = [4, 4, 4]
			maxCooldown = [1000, 800, 600]
			maxCast = [80, 60, 40]
			maxRange = [10, 12, 15]
			maxDiameter = [0.5, 0.5, 0.5]
			maxDamage = [4, 5, 6]
			maxVelocity = [0.5, 1.0, 1.5]

		[branches.diamond_branch.scatter_magic_ability]
			maxCost = [4, 4, 4]
			maxCooldown = [1000, 800, 600]
			maxCast = [80, 60, 40]
			maxRange = [3.0, 4.0, 5.0]
			maxDiameter = [0.5, 0.5, 0.5]
			maxDamage = [0.5, 1.0, 1.5]
			maxVelocity = [1.0, 1.5, 2.0]
			maxQuantity = [9, 13, 17]

		[branches.diamond_branch.rouse_magic_ability]
			maxCost = [4, 4, 4]
			maxCooldown = [1000, 800, 600]
			maxDuration = [40, 20, 10]
			maxCast = [80, 60, 40]
			maxRange = [6.0, 7.0, 9.0]
			maxDiameter = [3.0, 4.0, 5.0]
			maxDamage = [1.0, 1.5, 2.0]
			maxVelocity = [0.0, 0.0, 0.0]

		[branches.diamond_branch.ancient_ritual_ability]
			maxCost = [8, 6, 4]
			maxCooldown = [1000, 800, 600]
			maxDuration = [1200, 2400, 3600]

	[branches.bone_branch]

		[branches.bone_branch.leech_ability]
			maxCost = [6, 6, 4]
			maxCooldown = [2600, 1200, 900]
			maxDuration = [80, 120, 160]
			maxCharges = [1, 2, 3]
			maxLifeSteal = [0.4, 0.7, 1]

		[branches.bone_branch.bargain_ability]
			maxCost = [4, 4, 4]
			maxCooldown = [1000, 800, 600]
			maxDuration = [10, 10, 10]
			maxCast = [80, 60, 40]
			maxRange = [10, 15, 20]
			maxDiameter = [0.5, 0.5, 0.5]
			maxDamage = [0.5, 1.5, 2.5]
			maxVelocity = [0.5, 0.7, 1.0]
			maxHealing = [0.5, 1.5, 2.5]

		[branches.bone_branch.ghastling_ability]
			maxCost = [4, 4, 4]
			maxCooldown = [1000, 800, 600]
			maxDuration = [800, 1000, 1200]
			maxRange = [10, 12, 15]

		[branches.bone_branch.soulbound_ability]
			maxCost = [4, 4, 4]
			maxCooldown = [4800, 3300.0, 2100.0]
			maxDuration = [200, 260, 400]
			maxCast = [20, 20, 20]
			maxRange = [20, 30, 40]
			maxDiameter = [0.5, 0.5, 0.5]
			maxVelocity = [1.0, 2.0, 3.0]

	[branches.feather_branch]

		[branches.feather_branch.precise_shot_ability]
			maxCost = [6, 4, 2]
			maxCooldown = [320, 280, 200]
			maxDuration = [40, 60, 80]
			maxCharges = [1, 1, 1]
			maxVelocity = [0.4, 0.5, 0.7]

		[branches.feather_branch.agile_hands_ability]
			maxCost = [10, 8, 6]
			maxCooldown = [400, 300, 200]
			maxDuration = [40, 60, 80]
			maxCharges = [1, 2, 3]

		[branches.feather_branch.slide_hook_ability]
			maxCost = [8, 6, 2]
			maxCooldown = [500, 400, 320]
			maxDuration = [40, 60, 80]
			maxCharges = [1, 1, 2]
			maxVelocity = [2.0, 2.5, 3.5]

		[branches.feather_branch.hunter_hints_ability]
			maxCost = [6, 4, 2]
			maxCooldown = [400, 360, 320]
			maxDuration = [100, 200, 400]
			minMobs = [1, 2, 3]

	[branches.leaf_branch]

		[branches.leaf_branch.equilibrate_ability]
			maxCost = [8, 6, 4]
			maxCooldown = [400, 360, 320]
			maxCast = [80, 60, 40]
			maxRange = [8, 10, 15]
			maxDiameter = [0.5, 0.5, 0.5]
			maxVelocity = [1.0, 1.5, 2.0]
			maxEffectDuration = [160, 200, 240]

		[branches.leaf_branch.imitate_ability]

			[branches.leaf_branch.imitate_ability.player]
				maxCost = [8, 6, 4]
				maxCooldown = [900, 700, 500]
				maxDuration = [20, 10, 5]
				maxCast = [80, 60, 40]
				maxRange = [6, 8, 10]
				maxDiameter = [5.0, 7.0, 8.0]
				maxVelocity = [0.0, 0.0, 0.0]
				maxEffectDuration = [80, 100, 120]

			[branches.leaf_branch.imitate_ability.overgrowth_ability]
				maxCost = [12, 10, 8]
				maxCooldown = [400, 360, 320]
				maxDuration = [80, 60, 40]
				maxCharges = [1, 1, 1]
				maxRange = [1, 1, 2]

			[branches.leaf_branch.imitate_ability.morph_ability]
				maxCost = [12, 10, 8]
				maxCooldown = [400, 360, 320]
				maxDuration = [700, 900, 2400]

```

**Comment by dainxt** [comment]
If that error generated the first time then is normal. It is just the server trying to generate the configuration. If not, do you noticed some unusual behaviour inside the mod or the error throws on the next server starts?

**Comment by APaskowski1994** [comment]
Looks like the mod behaves normal.
But the error is trown constantly inside the console

**Comment by dainxt** [comment]
Probably fixed on 0.9.4, Happy Holidays and New Year!. Let me know if it is solved. You'll have to start once the server and restart to take effect.
